sbc_email/models/import_letters_history.py

```diff
@@ -151,6 +151,81 @@ def button_import(self):
 
             return super(ImportLettersHistory, self).button_import()
 
+    @api.multi
+    def button_save(self):
+        """
+        When saving the import_line as correspondences, move pdf file in the
+        done folder on the NAS and remove image attachment (Web letter case)
+        """
+        for letter in self:
+            if letter.config_id.name == "Web Letter":
```

Comment:
Would be better to check `if letter.config_id == self.env.ref('sbc_email.web_letter')`